styles/App.css

```diff
@@ -1,3 +1,409 @@
+
+    .button {
+        font-size: 2rem;
+        font-weight: 600;
+    }
+
+    #currentGameMode {        
+        font-size: 2.5rem;
+    }
+
+    #changeGameMode {
+        margin-bottom: 6rem;
+    }
+
+    .gameModeButton{
+        font-size: 3rem;
+    }
+
+}
+
+
+@media only screen and (min-width: 768px) and (max-width: 991px) {
+    #logo {
+        width: 25rem;
+        height: 14rem;
+    }
+
+    #img {
+        height: 34.375rem;
+        width: 25rem;
+    }
+
+    .button {
+        font-size: 2rem;
+        font-weight: 600;
+    }
+
+    #currentGameMode {  
+        padding: 2rem;      
+        font-size: 2rem;
+    }
+
+    #changeGameMode {
+        padding: 2rem;
+        margin-bottom: 4rem;
+    }
+
+    .gameModeButton{
+        font-size: 2rem;
+    }
+}
+
+
+@media only screen and (min-width: 576px) and (max-width: 767px) {
+
+    body {
+        padding: 0.5rem;
+        /* margin auto auto; */
+    }
+
+    #settingsButton {
+        order:2;
+    }
+
+    #img {
+        order:3;
+        position: absolute;
+        top: 54rem;
+    }
+
+    #changeGameMode {
+        margin-bottom: 42rem;
+        padding: 2rem;
+    }
+
+    .gameArea {
+        width: 100%;
+    }
+
+    #logo {
+        --width: 45rem;
+        width: var(--width);
+        height: calc(var(--width) * 0.56);
+    }
+
+    #img {
+        --width: 25rem;
+        width: var(--width);
+        height: calc(var(--width) * 1.495);
+    }
+
+    #currentGameMode {  
+        padding: 1.5rem;      
+        font-size: 2.5rem;
+    }
+
+    #currentGameModeDescription p {
+        font-size: 2rem;
+        padding: 0.5rem 0.5rem;
+    }
+
+    #currentGameModeDescription * {
+        margin: 0.5rem auto;
+    }
+
+    .button {
+        font-size: 2.2rem;
+        font-weight: 600;
+    }
+}
+
+@media only screen and (max-width: 575px) {
+    body {
+        padding: 0.5rem;
+    }
+
+    #settingsButton {
+        order:2;
+    }
+    
+    #img {
+        order:3;
+        position: absolute;
+        top: 62rem;
+    }
+
+    #changeGameMode {
+        flex-direction: column;
+        margin-bottom: 42rem;
+        padding: 2rem;
+    }
+
+    .gameArea {
+        width: 100%;
+    }
+
+    #logo {
+        --width: 30rem;
+        width: var(--width);
+        height: calc(var(--width) * 0.56);
+    }
+    
+    #img {
+        --width: 25rem;
+        width: var(--width);
+        height: calc(var(--width) * 1.495);
+    }
+
+    #currentGameMode {  
+        padding: 1.5rem;      
+        font-size: 2.5rem;
+    }
+
+    .gameModeButton:hover::before{
+        left:25%;
+        width:50%;
+    }
+
+    #currentGameModeDescription * {
+        margin: 0.5rem auto;
+    }
+
+    #currentGameModeDescription p {
+        font-size: 2rem;
+        padding: 0.5rem 0.5rem;
+    }
+
+    .button {
+        font-size: 2.2rem;
+        font-weight: 600;
+    }
+
+    #actionButtons .button{
+        width: 48%;
+        margin: 1rem 1rem;
+    }
+}
```

Comment:
this file is too long, I think it's necessary to cut it into small css files dedicated for components